pkgs/development/tools/quick-lint-js/default.nix

```diff
@@ -0,0 +1,26 @@
+{ cmake, fetchFromGitHub, lib, ninja, stdenv }:
+
+stdenv.mkDerivation rec {
+  pname = "quick-lint-js";
+  version = "2.1.0";
+
+  src = fetchFromGitHub {
+    owner = "quick-lint";
+    repo = "quick-lint-js";
+    rev = "${version}";
+    sha256 = "sha256-F21eli4HdLw3RComvocwBrcGfruIjO23E6+7a4+6vbs=";
+  };
```

Comment:
I don't know whether tarballs or GitHub is preferred in Nixpkgs. If tarballs are preferred, you can use the official release tarball with its accompanying GPG signature: https://c.quick-lint-js.com/releases/2.1.0/source/quick-lint-js-2.1.0.tar.gz and https://c.quick-lint-js.com/releases/2.1.0/source/quick-lint-js-2.1.0.tar.gz.asc

Reply:
Hmm, to be honest, I haven't seen a package use GPG signature :thinking:

Reply:
Some update scripts verify it but generally we prefer GitHub repositories because the tarballs are guaranteed to be correct and it is easier.